radar-upload-backend/src/main/java/org/radarbase/upload/dto/RecordMapperImpl.kt

```diff
@@ -46,7 +44,27 @@ class RecordMapperImpl: RecordMapper {
             modifiedDate = metadata.modifiedDate,
             committedDate = metadata.committedDate,
             logs = metadata.logs?.let {
-                LogsDto(url = "${uri.baseUri}/records/${metadata.id}/logs")
+                LogsDto(url = "${uri.baseUri}/records/${metadata.record.id}/logs")
             }
     )
+
+    override fun toMetadata(metadata: RecordMetadataDTO, origin: RecordMetadata) = RecordMetadata()
+            .also {
```

Comment:
Kotlin tip: `apply` gives the connotation that you will apply some changes to the RecordMetadata. It also means you do not have to write `it.` before every setter.